annif/static/css/style.css

```diff
@@ -1,2 +1,105 @@
-header { background-color: #3f9a88; }
-h1 { color: white; font-size: 2rem; }
+body {
+  background-color: white;
+  color: #343260;
+  font-family: Jost, sans-serif;
+  font-weight: 400;
+  font-size: 1.2rem;
+  line-height: 1.2
+}
+header {
+  background: #ffffff;
+  background: -moz-linear-gradient(top, #ffffff 92%, #d9dfe3 92%, #ffffff 100%);
+  background: -webkit-linear-gradient(top, #ffffff 92%, #d9dfe3 92%, #ffffff 100%);
+  background: linear-gradient(to bottom, #ffffff 92%, #d9dfe3 92%, #ffffff 100%);
+  padding-bottom: 5px;
+}
+a { color: #343260; text-decoration: underline; }
+a:hover, a:active { color: #6280dc; }
+a:visited { color: #536f9f }
+h1 { font-size: 2rem; }
+#form {
+  padding: 2rem;
+  background-color: #f3f3f6;
+  background: -moz-linear-gradient(top, #ffffff 0%, #d9dfe3 1%, #f3f3f6 1%, #f3f3f6 99%, #d9dfe3 99%, #ffffff 100%);
+  background: -webkit-linear-gradient(top, #ffffff 0%, #d9dfe3 1%, #f3f3f6 1%, #f3f3f6 99%, #d9dfe3 99%, #ffffff 100%);
+  background: linear-gradient(left, #ffffff 0%, #d9dfe3 1%, #f3f3f6 1%, #f3f3f6 99%, #d9dfe3 99%, #ffffff 100%);
+  background: linear-gradient(to right, #ffffff 0%, #d9dfe3 1%, #f3f3f6 1%, #f3f3f6 99%, #d9dfe3 99%, #ffffff 100%);
+  background: linear-gradient(to bottom, #ffffff 0%, #d9dfe3 1%, #f3f3f6 1%, #f3f3f6 99%, #d9dfe3 99%, #ffffff 100%);
```

Comment:
I think the first two `-moz-` and `-webkit` are browser-specific. Not too sure about the other three. Aren't they replacing each other, and only the last one is in effect?

Reply:
You're right. The browser-specific ones are hardly needed anymore and the others are just unnecessary duplicates